src/api/courses.js

```diff
@@ -4,13 +4,87 @@ const router = require('express').Router({
 
 const { check } = require('express-validator/check')
 const { matchedData } = require('express-validator/filter')
+const moment = require('moment')
 
-const { Course, Queue, User } = require('../models')
+const { Course, Queue, Question, User, Sequelize } = require('../models')
 const { requireCourse, requireUser, failIfErrors } = require('./util')
 const requireAdmin = require('../middleware/requireAdmin')
 const requireCourseStaff = require('../middleware/requireCourseStaff')
 const safeAsync = require('../middleware/safeAsync')
 
+// Helper for getting course question data
+const getColumns = questions => {
```

Comment:
It's a bit unclear to me what this function does - could you add a comment describing specifically what it returns?